### ISSUE TITLE
Browsertitel

### ISSUE THREAD

**Issue by muqit-zoarder** [GDI-HE/GeoPortal.rlp#47]
### Prüfschritt 2.4.2.a Sinnvolle Dokumenttitel

> BITV/WCAG-Prüfschritt: „Dokumenttitel sind eindeutig, sie bezeichnen Site und Seiteninhalt.“

**1. Problem:** Problem: Der Seitentitel „Geoportal“ gibt keine Auskunft über das Thema der Unterseite des Webauftritts

**LBIT:** Vorschlag LBIT: Die Seiten sind anhand des Seitentitels nicht unterscheidbar. Fehlerhafte Titel erschweren Nutzerinnen und Nutzern assistiver Technologie die Navigation zwischen verschiedenen Browser-Tabs und ebenso das Wiederfinden gespeicherter Lesezeichen, da diese oft automatisch vom Browser den Seitentitel als Namen erhalten. Der Titel muss aus einem statischen Teil (Seitenname) und einem dynamischen Teil (Thema der Unterseite) bestehen. Der Seitentitel muss angepasst werden, z. B. „Geoportal Hessen - Startseite“, „Geoportal Hessen - Kontakt“.

**Comment by karlbrink** [comment]
@muqit-zoarder 
the {{ wiki_keyword }} is not working - s.th. missing?
